src/infrastructure/NodeService.js

```diff
@@ -183,8 +183,6 @@ class NodeService {
 				// Api status
 				formattedNode.apiStatus = {
 					connectionStatus: isAvailable,
-					databaseStatus:
-						'up' === nodeStatus?.db || Constants.Message.UNAVAILABLE,
 					apiNodeStatus:
```

Comment:
Could you tell me what is an API Node? Light vs full.

Reply:
`databaseStatus` and `apiNodeStatus` are from the `/node/health` endpoint.

 `/node/health` endpoint response:
```
{"status":{"apiNode":"up","db":"up"}}
```

API node is a full node (roles contain API) because roles contain API, which supports the `/node/health` endpoint.

**Light** rest does not support the `/node/health` endpoint.

Reply:
Yes, the Light has an API Node with fewer features.  Showing the API Node with an X tells the user it is not working.
I would show it with the green check mark. You could ask  @0x6861746366574  on Discord.

Reply:
Actually I just worry users confuse, they treat Light REST node as Full REST for development.

I have another idea. 
`Light`: Light Node with the check mark, condition (apiStatus.isAvailable && [1, 3, 5].includes(node.roles) )
`Full`: API node with the check mark (remain the same)